web-components/src/components/icons/ProfileIcon.tsx

```diff
@@ -0,0 +1,67 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+import { sxToArray } from '../../utils/mui/sxToArray';
+
+interface ProfileIconProps extends SvgIconProps {}
+
+export default function ProfileIcon({
+  sx,
+  ...props
+}: ProfileIconProps): JSX.Element {
+  return (
+    <SvgIcon
+      sx={[...sxToArray(sx)]}
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
+      <path
```

Comment:
I think we miss the previous tags that were here: 
```
        <g clip-path="url(#clip0_251_24054)">
          <g filter="url(#filter0_d_251_24054)">
```

The `filter` and the `clip-path` defined in the file are not used anymore.

Reply:
ok, i added those tags back and made the ID adjustments here:
aac423ed5976c4fdafb207091d4f663c76e169dd
thanks for catching that detail !